examples/example_helper.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'faraday'
+
+module ExampleHelper
+  def self.print_output(line)
+    puts line unless ENV.fetch('SUPPRESS_WEBHOOK_MESSAGE', nil) == 'true'
```

Comment:
Could `$stderr.puts` this -- would also simplify the need for setting this env var when piping output!

```suggestion
    $stderr.puts line unless ENV.fetch('SUPPRESS_WEBHOOK_MESSAGE', nil) == 'true'
```